### PR TITLE
Improve ext-ds stubs

### PR DESCRIPTION
**Pull request by enumag**
Trying to fix https://github.com/phpstan/phpstan/issues/5635

### CODE REVIEW

File: stubs/ext-ds.stub [hunk 1]
```diff
@@ -17,9 +17,9 @@ use UnderflowException;
 interface Collection extends Traversable, Countable, JsonSerializable
 {
 	/**
-	 * @return Collection<TKey, TValue>
+	 * @return static<TKey, TValue>
```
Comment (ondrejmirtes) [inline stubs/ext-ds.stub:20]: `static<TKey, TValue>` isn't really supported right now, but in  this case you can just do `static`.

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

File: stubs/ext-ds.stub (hunk 1 above)
Comment (enumag) [reply stubs/ext-ds.stub:20]: I'm just experimenting for now. I don't even understand the problem fully yet. ;-)

**Comment by enumag** [comment]
@ondrejmirtes Something is failing but it seems unrelated to my changes.

```
php bin/phpstan analyse -c conf/config.stubFiles.neon -l 8 tests/notAutoloaded/empty.php
Stub file D:\a\phpstan-src\phpstan-src\stubs\ReflectionAttribute.stub is added multiple times.
```

**Comment by ondrejmirtes** [comment]
Please rebase, this is fixed on master.

**Comment by ondrejmirtes** [comment]
I see it's passing on Ubuntu so this is fine! 👍

**Pull request merged**